todo_app/data/trello_items.py

```diff
@@ -0,0 +1,88 @@
+import os, requests
+from flask import request
+
+trello_url = os.getenv("BASE_URL")
+trello_key = os.getenv("API_KEY")
+trello_token = os.getenv("TOKEN")
+
+trello_board_id = os.getenv("BOARD_ID")
+brainstorm_idlist = os.getenv("BRAINSTORM_IDLIST")
```

Comment:
I'm guessing this was for testing, as it's not mentioned elsewhere and doesn't seem to be used?